ml/src/main/scala/ai/h2o/sparkling/ml/algos/H2OGridSearch.scala

```diff
@@ -77,10 +77,10 @@ class H2OGridSearch(override val uid: String)
       valid: Option[H2OFrame]): Map[String, Any] = {
     algo.getH2OAlgorithmParams() ++
       Map(
-        "nfolds" -> getNfolds(),
-        "fold_column" -> getFoldCol(),
-        "response_column" -> getLabelCol(),
-        "weights_column" -> getWeightCol(),
+        "nfolds" -> getAlgo().getNfolds(),
```

Comment:
aren't the params already available in `algo.getH2OAlgorithmParams()` ?

Reply:
yes, you are right, thanks!